index.js

```diff
@@ -37,8 +39,27 @@ class BabelCompiler {
     }, {});
 
     opts.sourceMap = !!config.sourceMaps;
-    if (!opts.presets) opts.presets = ['latest'];
-    if (opts.presets.length === 0) delete opts.presets;
+
+    // Include 'latest' preset only if:
+    //  * there's no defined presets in brunch-config.js
+    //  * there's no .babelrc file in root directory
+    //  * there's no 'babel' field in package.json
+    if (!opts.presets) {
+      const babelrcPath = path.resolve(config.paths.root, '.babelrc');
```

Comment:
Not sure, it is not in the public API afaik, but it may be worth exposing.

However, I don't see it blocking from merging this PR. Some time later.